docs/framework/configure-apps/file-schema/wcf/add-of-backuplist.md

```diff
@@ -53,4 +53,4 @@ Represents a configuration element that defines a backup endpoint element.
 |[\<routing>](../../../../../docs/framework/configure-apps/file-schema/wcf/routing.md)|Contains a list of endpoints that you would like the Routing Service to use in case the primary endpoint can't be reached.|  
   
 ## See Also  
- [System.ServiceModel.Routing.Configuration.BackupEndpointElement](assetId:///System.ServiceModel.Routing.Configuration.BackupEndpointElement?qualifyHint=False&amp;autoUpgrade=True)
\ No newline at end of file
+ [<xref:System.ServiceModel.Routing.Configuration.BackupEndpointElement> 
```

Comment:
this should include "?displayProperty=fullName" I think that the opening bracket needs to be removed as well.